clients/fidesui/src/palette/palette.module.scss

```diff
@@ -0,0 +1,85 @@
+$colors: (
```

Comment:
With all the different libraries, utilities, etc having a single source for colors is going to be much simpler. This is great, and the added TS export of colors is just *chef kiss*.